### PR TITLE
Tt2 942 write initial processes on handling gdpr requests - CLI refactor part 1

### PR DESCRIPTION
**Pull request by zaydgds**
part of this process introduces a new CLI tool, bring the number of cli tools/helper scripts to 3. each tool was its own separate script so part of this is refactoring all the helper scripts into a single `yarn cli` command

This is part 1 of many which converts the script that sends the raw audit data into the single `yarn cli` command

This will be one of many pr's. im going for several small pr's so its easier to review

> [!IMPORTANT]  
> functionality of sending raw audit data has not changed. just how we call the script has

### CODE REVIEW

**Comment by katiewhitlock** [comment]
You've committed yarn cache files

File: scripts/manualAuditDataRequests/sendResults/sendAuditDataAction.ts
Comment (katiewhitlock) [inline scripts/manualAuditDataRequests/sendResults/sendAuditDataAction.ts:1]: Am sure you'd get to it in a subsequent request but be sure to update the script instructions in the README.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
+  - [Running the Integration Tests](#running-the-integration-tests)
+  - [Running feature tests against a feature branch in the Dev environment](#running-feature-tests-against-a-feature-branch-in-the-dev-environment)
+  - [Test Reports](#test-reports)
+  - [Creating and approving a Zendesk ticket](#creating-and-approving-a-zendesk-ticket)
+  - [Running Zendesk webhook locally](#running-zendesk-webhook-locally)
+- [Code standards](#code-standards)
+- [Scripts](#scripts)
+  - [Valid email recipients management](#valid-email-recipients-management)
+  - [Scripts for Raw Audit Data requests](#scripts-for-raw-audit-data-requests)
+    - [Starting the data copy to the Analysis bucket process](#starting-the-data-copy-to-the-analysis-bucket-process)
```
Comment (katiewhitlock) [inline README.md:18]: Is the script in here not outdated now?

File: scripts/cli.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import { sendAuditDataAction } from './manualAuditDataRequests/sendResults/sendAuditDataAction'
+import { AWS_REGION } from './utils/constants'
+
+process.env.AWS_REGION = AWS_REGION
+
+program
+  .name('ticf-integration-cli')
+  .description('cli tool for txma ticf integration')
+
+program
```
Comment (katiewhitlock) [inline scripts/cli.ts:11]: Adding prompts to these scripts could be useful, or an example to copy and paste from the readme

File: scripts/cli.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { Option, program } from '@commander-js/extra-typings'
+import { sendAuditDataAction } from './manualAuditDataRequests/sendResults/sendAuditDataAction'
+import { AWS_REGION } from './utils/constants'
+
+process.env.AWS_REGION = AWS_REGION
+program
+  .name('ticf-integration-cli')
+  .description('cli tool for txma ticfontegration')
```
[the remark this answers is not in the document]
Comment (zaydgds) [reply scripts/cli.ts]: fixed

File: README.md (hunk 1 above)
Comment (zaydgds) [reply README.md:18]: updated docs

File: scripts/cli.ts
Comment (katiewhitlock) [inline scripts/cli.ts:1]: The filename is a little vague. Could you try something like cliScriptSuite or cliUtils.

File: scripts/manualAuditDataRequests/sendResults/sendAuditDataAction.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+export const sendAuditDataAction = async (options: options) => {
+  copyManualRequestData(options.environment, options.athenaQueryId)
+    .then(() =>
+      console.log(
+        'Successfully copied data to automated queries folder, within Athena output bucket'
+      )
+    )
+    .catch((error: unknown) => {
+      console.error('Failed to copy data within output bucket', error)
```
Comment (katiewhitlock) [inline scripts/manualAuditDataRequests/sendResults/sendAuditDataAction.ts:20]: what do you mean by within output bucket?

File: .eslintrc.json [hunk 5]
```diff
@@ -9,7 +9,10 @@
     "jest/globals": true,
     "node": true
   },
-  "rules": { "no-console": "error" },
+  "rules": {
+    "no-console": "error",
+    "no-unused-vars": ["error", { "argsIgnorePattern": "^_*" }]
```
Comment (katiewhitlock) [inline .eslintrc.json:14]: why have you implemented this? Is it outlined in coding standards? Are there any instances in the code where we use unused variables intentionally?

Comment (zaydgds) [reply .eslintrc.json:14]: that is part of a change that im not ready to commit, its.. complicated to explain without code so i'll just remove the change

**Review by katiewhitlock — APPROVED** [review]

**Pull request merged**